src/components/rich-text-editor/rich-text-editor-write.ts

```diff
@@ -0,0 +1,30 @@
+import Component from 'vue-class-component';
+import { Prop, Vue } from 'vue-property-decorator';
+
+import { ModulVue } from '../../utils/vue/vue';
+import VueFroala from './adapter';
+import { MRichTextEditorOptions } from './rich-text-editor';
+import WithRender from './rich-text-editor-write.html';
+
+require('@fortawesome/fontawesome');
+require('@fortawesome/fontawesome-free-solid');
+
+require('froala-editor/js/froala_editor.pkgd.min');
+require('froala-editor/css/froala_editor.pkgd.min.css');
+
+Vue.use(VueFroala);
```

Comment:
You can register a component from inside the @Component annotation. This will create a local registration instead of a global registration.

Reply:
Yup I was getting there.